content/docs/glossary/muxed-accounts.mdx

```diff
@@ -272,4 +273,81 @@ If we were to make a payment between two muxed accounts that had *different* und
+
+In all cases, the action specified by the operation (be it a payment, clawback, etc.) will act as though the muxed address is the underlying account ID. Muxed accounts are not part of the network, so if your transaction succeeds, the operation will succeed. In the case of payments, then, **assets will always transfer**, regardless of sender/receiver support for muxed accounts.
+
+### Sending to the wrong account
+
+There are a number of ways to send data incorrectly, so these are grouped into a subcategory.
+
+Below, the "recipient" is defined as be the platform the destination account uses to interact with the Stellar network.
+
+#### Your operations use muxed accounts, but the recipients do not support muxed accounts.
+In general, you should not send muxed payments to people on platforms that do not support them. The platform will throw an exception trying to parse the relevant operation. For example, the JavaScript SDK will throw a helpful message:
+
+> destination is invalid; did you forget to enable muxing?
+
+Note, however, that the **operation will succeed** on the network. In the case of payments, for example, the destination's _parent address_ will still receive the funds.
+
+#### You want to send to a muxed account, but your platform does not support them.
+In this case, do not use a muxed address. The platform will likely fail to create the operation. You should fall back to the legacy method of using a transaction memo, instead.
+
+### Your platform receives a transaction with muxed destination account(s) _and_ a memo ID.
+
+In an ideal world, this situation would never happen. You can determine whether or not the underlying IDs are equal; if they aren't, this is a malformed transaction and we recommend not submitting this transaction.
+
+### Using a muxed address with an incompatible operation
+Only certain operations enable muxed accounts, as described [above][supported-ops]. Passing a muxed account to an incompatible parameter with an up-to-date SDK _should_ result in a compilation or runtime error at the time of use.
+
+For example, when using the JavaScript SDK incorrectly:
+
+<CodeExample>
+
+```js
+  const mAddress = "MA7QYNF7SOWQ3GLR2BGMZEHXAVIRZA4KVWLTJJFC7MGXUA74P7UJUAAAAAAAAAABUTGI4";
+  transactionBuilder.addOperation(
+    Operation.setTrustLineFlags({
+      trustor: mAddress, // wrong!
+      asset: Asset.native(),
+      flags: { clawbackEnabled: false }
+    })
+  );
+```
+
+</CodeExample>
+
+the runtime result would be:
+
+> Error: invalid version byte. expected 48, got 96
+
+This error message indicates that the `trustor` failed to parse as an account ID (`G...`). In other words, your code will fail and the invalid operation will never reach the network.
+
+### Using a muxed address without enabling the feature flag
```

Comment:
💭 We're about to enable the feature flag by default, so maybe we don't need this section? See https://stellarfoundation.slack.com/archives/C020KGHM5KQ/p1641402959045800

Reply:
We may want to keep this in in case someone is using an old version of an SDK.  But I think we should acknowledge that by adding a bit of copy that says something like "In up-to-date versions of Stellar SDKs, muxed accounts are on by default, but if you are using an earlier version of an SDK, they may still be behind a feature flag.  Our first  recommendation if you get this error is to install the newest version of any and all Stellar SDKs you use.  However, if that's not possible for some reason, you can will need to enable the feature flag before interacting with muxed accounts."

Reply:
Good idea, done!